syntax = "proto3";

package protodb.collection;

message ListCollectionsRequest {
    string database = 1;
}

message ListCollectionsResponse {
    enum FailureCode {
        NO_ERROR = 0;
        INTERNAL_ERROR = 1;
        INVALID_DATABASE = 2;
    }
    FailureCode failure_code = 1;

    repeated string collections = 2;
}